scripts/geopmpy/launcher.py

```diff
@@ -304,6 +304,9 @@ class Launcher(object):
     Abstract base class for MPI job launch application abstraction.
     Defines common methods used by all Launcher objects.
     """
+
+    is_slurm_enabled = False
```

Comment:
```suggestion
    _is_slurm_enabled = False
```
To denote this as a static member of the class.  There are very few examples of this in the codebase anymore, but this is how ```_is_once``` is defined for the IMPIExecLauncher.

Reply:
Done. FWIW as I understand it, this is to indicate that a member is protected, which is completely suitable.